src/components/aria-live/aria-live.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Component,
+  ElementRef,
+  AfterContentInit
+} from 'angular2/core';
+
+import {DOM} from '../../core/platform/dom/dom_adapter';
+
+@Component({
+  selector: 'md-aria-live',
+  templateUrl: './components/aria-live/aria-live.html',
+  styleUrls: ['./components/aria-live/aria-live.css'],
+})
+export class MdAriaLive implements AfterContentInit {
+
+  private announcerEl: HTMLElement;
+
+  constructor(private elementRef: ElementRef) { }
+
+  ngAfterContentInit() {
+    this.announcerEl = DOM.querySelector(this.elementRef.nativeElement, '._md-aria-announcer');
+  }
+
+  announce(message: string): void {
```

Comment:
It would be good if the announce function took a second, optional argument for the politeness.